### PR TITLE
clio-39828: SelectableCollection component in a working state

### PR DESCRIPTION
**Pull request by JefCurtis**

### CODE REVIEW

File: themis_components/thSelectableCollection/examples/1 - Simple/coffee.coffee [hunk 1]
```diff
@@ -0,0 +1,222 @@
+angular.module 'thDemo', ['ThemisComponents']
```
Comment (seanhealy) [inline themis_components/thSelectableCollection/examples/1 - Simple/coffee.coffee]: Ignore this.

File: themis_components/thSelectableCollection/examples/1 - Simple/coffee.coffee [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+  generateName = ->
+    possible = 'abcdefghijklmnopqrstuvwxyz'
+    length = getRandomInt 5, 10
+    text = ''
+    text += possible[getRandomInt 0, possible.length - 1] for i in [1 .. length]
+    capitalizeFirstLetter text
+
+  people = []
+  for i in [1 .. length]
```
Comment (seanhealy) [inline themis_components/thSelectableCollection/examples/1 - Simple/coffee.coffee]: This too. (Ignore)

File: themis_components/thSelectableCollection/examples/1 - Simple/coffee.coffee (hunk 1 above)
Comment (JefCurtis) [inline themis_components/thSelectableCollection/examples/1 - Simple/coffee.coffee]: Fixed!

File: themis_components/thSelectableCollection/examples/1 - Simple/coffee.coffee (hunk 2 above)
Comment (JefCurtis) [inline themis_components/thSelectableCollection/examples/1 - Simple/coffee.coffee]: Fixed!

**Pull request closed without merging**